www/front_src/src/App.tsx

```diff
@@ -11,40 +11,47 @@ import { connect } from 'react-redux';
 import { ConnectedRouter } from 'connected-react-router';
 import Fullscreen from 'react-fullscreen-crossbrowser';
 import queryString from 'query-string';
-import { batchActions } from 'redux-batched-actions';
 import { StylesProvider } from '@material-ui/styles';
-import Header from './components/header';
-import { history } from './store';
+import Header from './components/header/index.tsx';
```

Comment:
we shouldn't have to specify index files specifically

Reply:
Linter complains about it...